pkg/controller/issueapi/issue.go

```diff
@@ -254,6 +255,14 @@ func (c *Controller) HandleIssue() http.Handler {
 			return
 		}
 
+		err = c.recordCapacity(ctx, realm)
+		if err != nil {
+			logger.Errorw("failed to record realm capacity", "error", err)
+			stats.Record(ctx, c.metrics.CodeIssueErrors.M(1))
```

Comment:
Is this the right stat?

Reply:
Capacity recording does not return an error anymore.